### PR TITLE
[ Feature ] :: svg widget 

### LINKED ISSUE #2804: Feature :: Svg widget
### Is your proposal related to a problem?
Widget to view svg from svg code.

### Describe the solution you'd like
Build a new widget which takes in code as property and render svg.

### Describe alternatives you've considered
nothing

### If the feature is approved, would you be willing to submit a PR?
Yes.

### PR DESCRIPTION
**Pull request by stepinfwd**
resolves #2804

### CODE REVIEW

File: frontend/src/Editor/Components/components.js [hunk 1]
```diff
@@ -1979,4 +1979,43 @@ export const componentTypes = [
...
+      value: {},
+    },
+    definition: {
+      others: {
+        showOnDesktop: { value: '{{true}}' },
+        showOnMobile: { value: '{{false}}' },
+      },
+      properties: {
+        data: {
+          value: '',
```
Comment (Navaneeth-pk) [inline frontend/src/Editor/Components/components.js:2012]: Let's add a default svg here?

File: frontend/src/Editor/Components/components.js [hunk 2]
```diff
@@ -1979,4 +1979,43 @@ export const componentTypes = [
       },
     },
   },
+  {
+    name: 'SvgViewer',
+    displayName: 'Svg Viewer',
+    description: 'Svg viewer',
+    component: 'SvgViewer',
+    properties: {
+      data: { type: 'code', displayName: 'SvgViewer data' },
```
Comment (Navaneeth-pk) [inline frontend/src/Editor/Components/components.js:1988]: Can we rename the `displayName` to something better?

**Comment by Navaneeth-pk** [comment]
image

Contents of the widget should not overflow.

**Comment by stepinfwd** [comment]
updated @Navaneeth-pk

**Review by sherfin94 — APPROVED** [review]

**Pull request merged**